extensions/everhour/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "everhour",
+  "title": "Everhour",
```

Comment:
Maybe add a little bit more context here? The extension title is used in the store.
```suggestion
  "title": "Everhour Time Tracking",
```

Reply:
Good shout!